2021-10-02 23:44:58 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:75)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Desktop/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2021-10-02 23:45:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "PRODUCTO" no encontrada
Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2021-10-02 23:48:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "PRODUCTO" no encontrada
Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2021-10-03 18:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversi√≥n de datos fallida, convirtiendo "Cazarecompensas"
Data conversion error converting "Cazarecompensas" [22018-200]
2021-10-03 18:49:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "descripcion" no encontrada
Column "descripcion" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getDouble(JdbcResultSet.java:752)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2021-10-05 09:47:32 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:81)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2021-10-05 10:02:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "UPDATE NAVE SET DESCRIPCION = 'CUCU' WHERE IDNAVE 1[*]"
Syntax error in SQL statement "UPDATE NAVE SET DESCRIPCION = 'CUCU' WHERE IDNAVE 1[*]"; SQL statement:
UPDATE NAVE SET DESCRIPCION = 'CUCU' WHERE IDNAVE 1 [42000-200]
2021-10-05 10:04:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "descripcion" no encontrada
Column "descripcion" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.consultarPersonaje(Aplicacion.java:220)
	at ejemplo.Aplicacion.main(Aplicacion.java:117)
2021-10-05 10:05:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "descripcion" no encontrada
Column "descripcion" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.consultarPersonaje(Aplicacion.java:220)
	at ejemplo.Aplicacion.main(Aplicacion.java:117)
2021-10-05 10:06:26 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:81)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:D:/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2021-10-05 10:07:04 database: wrong user or password; user: "SA"
org.h2.message.DbException: Nombre de usuario √≥ password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario √≥ password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2021-10-19 19:12:39 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:85)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/profesorado/Dropbox/miParticularPROG/Tema10/UT10_ProyectoBaseH2/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2022-03-22 12:38:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Este metodo no esta permitido para una sentencia preparada; en su lugar use una sentencia regular.
This method is not allowed for a prepared statement; use a regular statement instead. [90130-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:322)
	at ejemplo.Aplicacion.borrarPersonaje(Aplicacion.java:327)
	at ejemplo.Aplicacion.main(Aplicacion.java:145)
2024-04-10 13:35:49 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:83)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-10 13:54:00 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:83)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/UT10_ProyectoBaseH2/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-12 12:16:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "NAVE[*]"
Syntax error in SQL statement "NAVE[*]"; SQL statement:
NAVE [42000-200]
2024-04-12 12:20:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "DATE" "2001/12/19"
Cannot parse "DATE" constant "2001/12/19"; SQL statement:
insert into Peliculas values ('111','El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001/12/19'); [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:215)
	at ejemplo.Aplicacion.main(Aplicacion.java:110)
Caused by: java.lang.NumberFormatException: For input string: "2001/1"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 16 more
2024-04-12 12:22:15 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "DATE" "2001/12/19"
Cannot parse "DATE" constant "2001/12/19"; SQL statement:
insert into Peliculas values ('111','El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001/12/19') [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2001/1"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2024-04-12 12:22:15 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "DATE" "2002/12/18"
Cannot parse "DATE" constant "2002/12/18"; SQL statement:
insert into Peliculas values ('112','El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002/12/18') [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2002/1"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2024-04-12 12:22:15 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "DATE" "2003/12/17"
Cannot parse "DATE" constant "2003/12/17"; SQL statement:
insert into Peliculas values ('113','El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003/12/17') [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "2003/1"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2024-04-12 12:44:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PELICULAS" debido a que "CONSTRAINT_C59, CONSTRAINT_1BD" depende de √©l.
Cannot drop "PELICULAS" because "CONSTRAINT_C59, CONSTRAINT_1BD" depends on it; SQL statement:
DROP TABLE IF EXISTS Peliculas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:186)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2024-04-12 12:45:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PELICULAS" debido a que "CONSTRAINT_C59, CONSTRAINT_1BD" depende de √©l.
Cannot drop "PELICULAS" because "CONSTRAINT_C59, CONSTRAINT_1BD" depends on it; SQL statement:
DROP TABLE IF EXISTS Peliculas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:186)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2024-04-12 12:46:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PELICULAS" debido a que "CONSTRAINT_C59, CONSTRAINT_1BD" depende de √©l.
Cannot drop "PELICULAS" because "CONSTRAINT_C59, CONSTRAINT_1BD" depends on it; SQL statement:
DROP TABLE IF EXISTS Peliculas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:186)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2024-04-12 12:55:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PELICULAS" debido a que "CONSTRAINT_C59, CONSTRAINT_1BD" depende de √©l.
Cannot drop "PELICULAS" because "CONSTRAINT_C59, CONSTRAINT_1BD" depends on it; SQL statement:
DROP TABLE IF EXISTS Peliculas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:186)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2024-04-12 12:58:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PELICULAS" debido a que "CONSTRAINT_C59, CONSTRAINT_1BD" depende de √©l.
Cannot drop "PELICULAS" because "CONSTRAINT_C59, CONSTRAINT_1BD" depends on it; SQL statement:
DROP TABLE IF EXISTS Peliculas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:186)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2024-04-12 13:15:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "DROP*[*] FROM PLATAFORMAS"; se esperaba "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"
Syntax error in SQL statement "DROP*[*] FROM PLATAFORMAS"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
drop* FROM PLATAFORMAS [42001-200]
2024-04-12 13:17:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PLATAFORMAS" debido a que "CONSTRAINT_1B" depende de √©l.
Cannot drop "PLATAFORMAS" because "CONSTRAINT_1B" depends on it; SQL statement:
DROP TABLE IF EXISTS Plataformas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:186)
	at ejemplo.Aplicacion.main(Aplicacion.java:107)
2024-04-12 13:37:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversi√≥n de datos fallida, convirtiendo "'ABCD12345' (ACTORES: ""CODIGO"" INT NOT NULL)"
Data conversion error converting "'ABCD12345' (ACTORES: ""CODIGO"" INT NOT NULL)"; SQL statement:


insert into Peliculas values (111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19'); [22018-200]
2024-04-12 13:38:32 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:83)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-12 13:38:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversi√≥n de datos fallida, convirtiendo "'ABCD12345' (ACTUACION: ""CODACTOR"" INT NOT NULL)"
Data conversion error converting "'ABCD12345' (ACTUACION: ""CODACTOR"" INT NOT NULL)"; SQL statement:


insert into Peliculas values (111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19'); [22018-200]
2024-04-12 14:18:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversi√≥n de datos fallida, convirtiendo "'ABCD12345' (ACTORES: ""CODIGO"" INT NOT NULL)"
Data conversion error converting "'ABCD12345' (ACTORES: ""CODIGO"" INT NOT NULL)"; SQL statement:
insert into Peliculas values (111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19'); [22018-200]
2024-04-12 14:19:47 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:83)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-12 14:20:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversi√≥n de datos fallida, convirtiendo "'ABCD12345' (ACTORES: ""CODIGO"" INT NOT NULL)"
Data conversion error converting "'ABCD12345' (ACTORES: ""CODIGO"" INT NOT NULL)"; SQL statement:
insert into Peliculas values (111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19'); [22018-200]
2024-04-15 12:46:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Imposible eliminar "PELICULAS" debido a que "CONSTRAINT_C59" depende de √©l.
Cannot drop "PELICULAS" because "CONSTRAINT_C59" depends on it; SQL statement:


DROP TABLE IF EXISTS Peliculas ; [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:59)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.crearTablas(Aplicacion.java:187)
	at ejemplo.Aplicacion.main(Aplicacion.java:108)
2024-04-15 12:49:31 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "CODPELICULA" no encontrada
Column "CODPELICULA" not found; SQL statement:
DELETE FROM Peliculas WHERE CODPELICULA=112 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-04-15 12:49:53 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "CODPELICULA" no encontrada
Column "CODPELICULA" not found; SQL statement:
DELETE FROM Peliculas WHERE CODPELICULA=113 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Delete.prepare(Delete.java:184)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-04-15 13:22:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SELECT CODPELICULA FROM DISPONIBLE_EN WHERE CODPLATAFORMA=%[*]D"; se esperaba "ALL, ANY, SOME"
Syntax error in SQL statement "SELECT CODPELICULA FROM DISPONIBLE_EN WHERE CODPLATAFORMA=%[*]D"; expected "ALL, ANY, SOME"; SQL statement:
SELECT codPelicula FROM Disponible_en WHERE codPlataforma=%d [42001-200]
2024-04-15 13:25:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SELECT CODPELICULA FROM DISPONIBLE_EN WHERE CODPLATAFORMA=%[*]D"; se esperaba "ALL, ANY, SOME"
Syntax error in SQL statement "SELECT CODPELICULA FROM DISPONIBLE_EN WHERE CODPLATAFORMA=%[*]D"; expected "ALL, ANY, SOME"; SQL statement:
SELECT codPelicula FROM Disponible_en WHERE codPlataforma=%d [42001-200]
2024-04-15 13:26:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:350)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:28:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:350)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:31:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:350)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:34:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:350)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:35:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:351)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:37:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:351)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:42:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:351)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:49:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "codPelicula" no encontrada
Column "codPelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:351)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:50:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SELECT CODIGO, NOMBRE FROM PELICULAS WHERE CODIGO=%[*]D"; se esperaba "ALL, ANY, SOME"
Syntax error in SQL statement "SELECT CODIGO, NOMBRE FROM PELICULAS WHERE CODIGO=%[*]D"; expected "ALL, ANY, SOME"; SQL statement:
SELECT codigo, nombre FROM Peliculas WHERE codigo=%d [42001-200]
2024-04-15 13:51:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "NOMBRE" no encontrada
Column "NOMBRE" not found; SQL statement:
SELECT codigo, nombre FROM Peliculas WHERE codigo=111 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:355)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 13:52:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "titulo" no encontrada
Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.consultarPeliculasEnPlataformas(Aplicacion.java:359)
	at ejemplo.Aplicacion.main(Aplicacion.java:140)
2024-04-15 14:16:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "titulo" no encontrada
Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:289)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-04-15 14:17:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "NAVE" no encontrada
Table "NAVE" not found; SQL statement:
UPDATE NAVE SET DESCRIPCION = 'HIPOPOTAMO' WHERE IDNAVE = ? [42102-200]
2024-04-15 14:28:08 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:84)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-15 14:28:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "TACHULON" no encontrada
Column "TACHULON" not found; SQL statement:
UPDATE Plataformas SET nombre = TAchulon WHERE codigo = 333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:304)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-04-15 14:31:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "TACHU" no encontrada
Column "TACHU" not found; SQL statement:
UPDATE Plataformas SET nombre = tachu WHERE codigo = 333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:304)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-04-15 14:35:56 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:84)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-15 14:36:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "tachu" no encontrada
Column "tachu" not found; SQL statement:
UPDATE Plataformas SET nombre = "tachu" WHERE codigo = 333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:304)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-04-15 14:36:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "tachu" no encontrada
Column "tachu" not found; SQL statement:
UPDATE Plataformas SET nombre = "tachu" WHERE codigo = 333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:304)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-04-15 14:37:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "tachyu" no encontrada
Column "tachyu" not found; SQL statement:
UPDATE Plataformas SET nombre = "tachyu" WHERE codigo = 333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:304)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-04-15 14:38:21 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "gfdgd" no encontrada
Column "gfdgd" not found; SQL statement:
UPDATE Plataformas SET nombre = "gfdgd" WHERE codigo = 333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-04-15 18:33:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Valor demasiado largo para la columna "URLLOGOTIPO VARCHAR(200)": "'https://www.google.com/url?sa=i&url=https%3A%2F%2Fes.wikipedia.org%2Fwiki%2FHBO&psig=AOvVaw1XhX5KDIxQgyRN1SjMka0H&ust=171328518... (214)"
Value too long for column "URLLOGOTIPO VARCHAR(200)": "'https://www.google.com/url?sa=i&url=https%3A%2F%2Fes.wikipedia.org%2Fwiki%2FHBO&psig=AOvVaw1XhX5KDIxQgyRN1SjMka0H&ust=171328518... (214)"; SQL statement:
insert into Peliculas values (111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19'); [22001-200]
2024-04-16 17:24:44 database: flush
org.h2.message.DbException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ejemplo.Aplicacion.main(Aplicacion.java:84)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:G:/Mi unidad/NetBeansProjects/PROG10_Tarea2023_24_Solucion/proyectobase.h2db.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 19 more
2024-04-29 18:22:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SINOPSIS" not found; SQL statement:

-- Datos para tabla Peliculas
MERGE INTO Peliculas(codigo, titulo, sinopsis, fEstreno) VALUES

 111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19';
-- 112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
-- 113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
-- 114,'Matrix','Neo descubre la verdad', '2003-12-04');
-- 115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
-- 116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
-- 117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
-- 118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseMerge(Parser.java:1618)
	at org.h2.command.Parser.parsePrepared(Parser.java:959)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:212)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2024-04-29 18:23:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SINOPSIS" not found; SQL statement:

-- Datos para tabla Peliculas
MERGE INTO Peliculas(codigo, titulo, sinopsis, fEstreno) VALUES

 111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19',
112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
-- 113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
-- 114,'Matrix','Neo descubre la verdad', '2003-12-04');
-- 115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
-- 116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
-- 117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
-- 118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseMerge(Parser.java:1618)
	at org.h2.command.Parser.parsePrepared(Parser.java:959)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:212)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2024-04-29 18:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
                             
MERGE INTO PELICULAS(CODIGO, TITULO, SINOPSIS, FESTRENO) VALUES

111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19',
112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18')[*];
                                                                                           
                                                        
                                                                         
                                                                        
                                                                                    
                                                                                                         


                               
                                                                                    
                                                                                                              
                                                                                                                                                                     
                                                                                                                                                            

                                  
                         
                         
                         
                         
                         
                         
                         
                         
                         
                         
                        '"; SQL statement:

-- Datos para tabla Peliculas
MERGE INTO Peliculas(codigo, titulo, sinopsis, fEstreno) VALUES

111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19',
112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
-- 113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
-- 114,'Matrix','Neo descubre la verdad', '2003-12-04');
-- 115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
-- 116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
-- 117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
-- 118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [42000-200]
2024-04-29 18:28:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:

-- Datos para tabla Peliculas
MERGE INTO Peliculas(CODIGO, TITULO, SINOPSIS, FESTRENO) VALUES
111,'El se√±or de los anillos: La comunidad del anillo','Frodo recibe el anillo de poder', '2001-12-19';
112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 114,'Matrix','Neo descubre la verdad', '2003-12-04');
115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [21002-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.dml.Merge.prepare(Merge.java:271)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at ejemplo.Aplicacion.insertarDatosTablas(Aplicacion.java:212)
	at ejemplo.Aplicacion.main(Aplicacion.java:111)
2024-04-29 18:29:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
112[*],'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 114,'Matrix','Neo descubre la verdad', '2003-12-04');
115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


                               
                                                                                    
                                                                                                              
                                                                                                                                                                     
                                                                                                                                                            

                                  
                         
                         
                         
                         
                         
                         
                         
                         
                         
                         
                        '"; SQL statement:

112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 114,'Matrix','Neo descubre la verdad', '2003-12-04');
115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [42000-200]
2024-04-29 18:30:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
(112[*],'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
(113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 (114,'Matrix','Neo descubre la verdad', '2003-12-04');
(115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 (116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 (117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 (118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


                               
                                                                                    
                                                                                                              
                                                                                                                                                                     
                                                                                                                                                            

                                  
                         
                         
                         
                         
                         
                         
                         
                         
                         
                         
                        '"; expected "(, WITH, SELECT, TABLE, VALUES"; SQL statement:

(112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
(113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 (114,'Matrix','Neo descubre la verdad', '2003-12-04');
(115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 (116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 (117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 (118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [42001-200]
2024-04-29 18:32:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
(112[*],'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
(113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 (114,'Matrix','Neo descubre la verdad', '2003-12-04');
(115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 (116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 (117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 (118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


                               
                                                                                    
                                                                                                              
                                                                                                                                                                     
                                                                                                                                                            

                                  
                         
                         
                         
                         
                         
                         
                         
                         
                         
                         
                        '"; expected "(, WITH, SELECT, TABLE, VALUES"; SQL statement:

(112,'El se√±or de los anillos: Las dos torres','Frodo pasa entre dos torres', '2002-12-18');
(113,'El se√±or de los anillos: El retorno del rey','Frodo tira el anillo', '2003-12-17');
 (114,'Matrix','Neo descubre la verdad', '2003-12-04');
(115,'Matrix Reloaded','Neo luchas contra las m√°quinas', '2005-12-05');
 (116,'Matrix Revollution','Neo acaba con las m√°quinas', '2006-12-04');
 (117,'Avenger civil war','Thanos va por las gemas y hace la purga', '2006-12-04');
 (118,'Avenger end game','Vuelven al pasado tras el multiverso para cambiar el presente', '2006-12-04');


-- Datos para tabla Plataformas
--'333','Netflix','https://images-na.ssl-images-amazon.com/images/I/51LGj5--KsL.png'
--'334','Movistar','https://www.tuexperto.com/wp-content/uploads/2018/09/80dd8ee71e2be472cf86ee124476966a.png'
--'335','Disney+','https://store-images.s-microsoft.com/image/apps.14187.14495311847124170.7646206e-bd82-4cf0-8b8c-d06a67bc302c.2e474878-acb7-4afb-a503-c2a1a32feaa8'
--'336','HBO max','https://yt3.googleusercontent.com/bjkwQJm7D-ynsH_4ySvk-sqM802R1RYuYDqTazAF-g3gupsidu981t-uA3FFzlkdkK5txkXsLjg=s900-c-k-c0x00ffffff-no-rj'

-- Datos para tabla Disponible_en 
-- 333, 111, '2018-01-12'
-- 333, 112, '2018-02-13'
-- 333, 115, '2018-03-14'
-- 333, 116, '2018-04-15'
-- 334, 112, '2020-05-02'
-- 334, 113, '2020-06-02'
-- 334, 114, '2020-07-02'
-- 334, 115, '2021-08-02'
-- 334, 116, '2021-09-02'
-- 335, 117, '2022-01-10'
-- 335, 118, '2022-05-02' [42001-200]
2024-04-29 18:50:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "2001-12-19" [22018-200]
2024-04-29 18:51:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "2001-12-19" [22018-200]
2024-04-29 18:52:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "2001-12-19" [22018-200]
2024-04-29 18:52:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "2001-12-19" [22018-200]
2024-04-29 18:53:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "2001-12-19" [22018-200]
2024-04-30 09:31:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-04-30 09:31:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT * FROM producto [42102-200]
2024-04-30 09:33:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-04-30 09:33:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-04-30 09:36:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-04-30 09:39:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-04-30 09:40:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-04-30 09:41:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-200]
2024-04-30 09:47:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUCTO" not found; SQL statement:
SELECT id,nombre,precio FROM producto [42102-200]
2024-05-01 11:03:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "codigo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:363)
	at ejemplo.Aplicacion.main(Aplicacion.java:143)
2024-05-01 11:16:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIO" not found; SQL statement:
DELETE FROM USUARIO WHERE CODIGO=? [42102-200]
2024-05-01 11:54:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "3" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:362)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-05-01 11:55:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "3" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:362)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-05-01 11:57:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "3" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:362)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-05-01 11:59:28 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "HOLA" not found; SQL statement:
UPDATE Plataformas SET NOMBRE=hola WHERE CODIGO=333 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Update.prepare(Update.java:275)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:833)
2024-05-01 12:00:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "3" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:362)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-05-01 12:01:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Invalid value "2" for parameter "parameterIndex" [90008-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1503)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:394)
	at ejemplo.Aplicacion.modificarPlataforma(Aplicacion.java:362)
	at ejemplo.Aplicacion.main(Aplicacion.java:141)
2024-05-01 12:36:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "codpelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:443)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:36:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "codpelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:443)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:36:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "codpelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:443)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:36:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "codpelicula" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:443)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:41:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:447)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:41:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:447)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:41:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:447)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:42:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODPELICULA" not found; SQL statement:
SELECT titulo FROM peliculas where codpelicula=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:444)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:42:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODPELICULA" not found; SQL statement:
SELECT titulo FROM peliculas where codpelicula=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:444)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:42:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODPELICULA" not found; SQL statement:
SELECT titulo FROM peliculas where codpelicula=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:444)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:43:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODPELICULA" not found; SQL statement:
SELECT titulo FROM peliculas where codpelicula=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:444)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:43:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODPELICULA" not found; SQL statement:
SELECT titulo FROM peliculas where codpelicula=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:444)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:43:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "CODPELICULA" not found; SQL statement:
SELECT titulo FROM peliculas where codpelicula=? [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:352)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:444)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:48:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:447)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:48:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:447)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-01 12:48:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "titulo" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3195)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:316)
	at ejemplo.Aplicacion.mostrarPeliculasPorPlataformas(Aplicacion.java:447)
	at ejemplo.Aplicacion.main(Aplicacion.java:144)
2024-05-06 17:46:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "

  
                           
  
CREATE TABLE IF NOT EXISTS PELICULAS (
CODIGO INT NOT NULL,
TITULO VARCHAR(200) NOT NULL,    
SINOPSIS VARCHAR(200)NOT NULL,   
FESTRENO DATE NOT NULL,
PRIMARY KEY  (CODIGO)NOT[*] NULL
);
CREATE TABLE IF NOT EXISTS PLATAFORMAS (
   CODIGO INT NOT NULL,
NOMBRE VARCHAR(200) NOT NULL ,    
URLLOGOTIPO VARCHAR(200) NOT NULL,
PRIMARY KEY  (CODIGO) NOT NULL
);
CREATE TABLE IF NOT EXISTS DISPONIBLE_EN (
CODPLATAFORMA INT NOT NULL,
CODPELICULA INT NOT NULL,
FDISPONIBILIDADDA DATE NOT NULL,
PRIMARY KEY (CODPLATAFORMA, CODPELICULA) NOT NULL,
FOREIGN KEY (CODPELICULA) REFERENCES PELICULAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
FOREIGN KEY (CODPLATAFORMA) REFERENCES PLATAFORMAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE NOT NULL

);"; expected "INDEX, ,, )"; SQL statement:


--
-- Estructura de las tablas
--
CREATE TABLE IF NOT EXISTS Peliculas (
codigo INT NOT NULL,
titulo VARCHAR(200) NOT NULL,    
sinopsis VARCHAR(200)NOT NULL,   
fEstreno DATE NOT NULL,
PRIMARY KEY  (codigo)NOT NULL
);
CREATE TABLE IF NOT EXISTS Plataformas (
   codigo INT NOT NULL,
nombre VARCHAR(200) NOT NULL ,    
urlLogotipo VARCHAR(200) NOT NULL,
PRIMARY KEY  (codigo) NOT NULL
);
CREATE TABLE IF NOT EXISTS Disponible_en (
codPlataforma INT NOT NULL,
codPelicula INT NOT NULL,
fDisponibilidadda DATE NOT NULL,
PRIMARY KEY (codPlataforma, codPelicula) NOT NULL,
FOREIGN KEY (codPelicula) REFERENCES Peliculas(codigo)ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
FOREIGN KEY (codPlataforma) REFERENCES Plataformas(codigo)ON UPDATE CASCADE ON DELETE CASCADE NOT NULL

); [42001-200]
2024-05-06 17:48:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE IF NOT EXISTS DISPONIBLE_EN (
CODPLATAFORMA INT NOT NULL,
CODPELICULA INT NOT NULL,
FDISPONIBILIDADDA DATE NOT NULL,
PRIMARY KEY (CODPLATAFORMA, CODPELICULA) NOT[*] NULL,
FOREIGN KEY (CODPELICULA) REFERENCES PELICULAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (CODPLATAFORMA) REFERENCES PLATAFORMAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE 

);"; expected "INDEX, ,, )"; SQL statement:

CREATE TABLE IF NOT EXISTS Disponible_en (
codPlataforma INT NOT NULL,
codPelicula INT NOT NULL,
fDisponibilidadda DATE NOT NULL,
PRIMARY KEY (codPlataforma, codPelicula) NOT NULL,
FOREIGN KEY (codPelicula) REFERENCES Peliculas(codigo)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (codPlataforma) REFERENCES Plataformas(codigo)ON UPDATE CASCADE ON DELETE CASCADE 

); [42001-200]
2024-05-06 17:48:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE IF NOT EXISTS DISPONIBLE_EN (
CODPLATAFORMA INT NOT NULL,
CODPELICULA INT NOT NULL,
FDISPONIBILIDADDA DATE NOT NULL,
PRIMARY KEY (CODPLATAFORMA, CODPELICULA)
FOREIGN[*] KEY (CODPELICULA) REFERENCES PELICULAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (CODPLATAFORMA) REFERENCES PLATAFORMAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE 

);"; expected "INDEX, ,, )"; SQL statement:

CREATE TABLE IF NOT EXISTS Disponible_en (
codPlataforma INT NOT NULL,
codPelicula INT NOT NULL,
fDisponibilidadda DATE NOT NULL,
PRIMARY KEY (codPlataforma, codPelicula)
FOREIGN KEY (codPelicula) REFERENCES Peliculas(codigo)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (codPlataforma) REFERENCES Plataformas(codigo)ON UPDATE CASCADE ON DELETE CASCADE 

); [42001-200]
2024-05-06 17:48:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE IF NOT EXISTS DISPONIBLE_EN (
CODPLATAFORMA INT NOT NULL,
CODPELICULA INT NOT NULL,
FDISPONIBILIDADDA DATE NOT NULL,
PRIMARY KEY (CODPLATAFORMA, CODPELICULA)
FOREIGN[*] KEY (CODPELICULA) REFERENCES PELICULAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (CODPLATAFORMA) REFERENCES PLATAFORMAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE 

);"; expected "INDEX, ,, )"; SQL statement:

CREATE TABLE IF NOT EXISTS Disponible_en (
codPlataforma INT NOT NULL,
codPelicula INT NOT NULL,
fDisponibilidadda DATE NOT NULL,
PRIMARY KEY (codPlataforma, codPelicula)
FOREIGN KEY (codPelicula) REFERENCES Peliculas(codigo)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (codPlataforma) REFERENCES Plataformas(codigo)ON UPDATE CASCADE ON DELETE CASCADE 

); [42001-200]
2024-05-06 17:49:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE IF NOT EXISTS PLATAFORMAS (
CODIGO INT  ,
NOMBRE VARCHAR(200)  ,    
URLLOGOTIPO VARCHAR(200) 
PRIMARY KEY  ([*]CODIGO) 
);
CREATE TABLE IF NOT EXISTS DISPONIBLE_EN (
CODPLATAFORMA INT NOT NULL,
CODPELICULA INT NOT NULL,
FDISPONIBILIDADDA DATE NOT NULL,
PRIMARY KEY (CODPLATAFORMA, CODPELICULA)
FOREIGN KEY (CODPELICULA) REFERENCES PELICULAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (CODPLATAFORMA) REFERENCES PLATAFORMAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE 

);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:

CREATE TABLE IF NOT EXISTS Plataformas (
codigo INT  ,
nombre VARCHAR(200)  ,    
urlLogotipo VARCHAR(200) 
PRIMARY KEY  (codigo) 
);
CREATE TABLE IF NOT EXISTS Disponible_en (
codPlataforma INT NOT NULL,
codPelicula INT NOT NULL,
fDisponibilidadda DATE NOT NULL,
PRIMARY KEY (codPlataforma, codPelicula)
FOREIGN KEY (codPelicula) REFERENCES Peliculas(codigo)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (codPlataforma) REFERENCES Plataformas(codigo)ON UPDATE CASCADE ON DELETE CASCADE 

); [42001-200]
2024-05-06 17:49:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "
CREATE TABLE IF NOT EXISTS PLATAFORMAS (
CODIGO INT  ,
NOMBRE VARCHAR(200)  ,    
URLLOGOTIPO VARCHAR(200) 
PRIMARY KEY  ([*]CODIGO) 
);
CREATE TABLE IF NOT EXISTS DISPONIBLE_EN (
CODPLATAFORMA INT  ,
CODPELICULA INT  ,
FDISPONIBILIDADDA DATE  ,
PRIMARY KEY (CODPLATAFORMA, CODPELICULA)
FOREIGN KEY (CODPELICULA) REFERENCES PELICULAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (CODPLATAFORMA) REFERENCES PLATAFORMAS(CODIGO)ON UPDATE CASCADE ON DELETE CASCADE 

);"; expected "HASH, AUTO_INCREMENT, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:

CREATE TABLE IF NOT EXISTS Plataformas (
codigo INT  ,
nombre VARCHAR(200)  ,    
urlLogotipo VARCHAR(200) 
PRIMARY KEY  (codigo) 
);
CREATE TABLE IF NOT EXISTS Disponible_en (
codPlataforma INT  ,
codPelicula INT  ,
fDisponibilidadda DATE  ,
PRIMARY KEY (codPlataforma, codPelicula)
FOREIGN KEY (codPelicula) REFERENCES Peliculas(codigo)ON UPDATE CASCADE ON DELETE CASCADE,
FOREIGN KEY (codPlataforma) REFERENCES Plataformas(codigo)ON UPDATE CASCADE ON DELETE CASCADE 

); [42001-200]
